res/openvpn-gui-res-kr.rc

```diff
@@ -0,0 +1,454 @@
+          "패킷 인증 및 패킷 압축을 지원하여 단일 TCP/UDP 포트를 통해 " \
+          "IP 네트워크를 안전하게 터널링하는 응용 프로그램 입니다.\n" \
+          "\n" \
+          "Copyright (C) 2002-2016 OpenVPN Technologies, Inc <info@openvpn.net>\n" \
+          "https://openvpn.net/", 0, 8, 61, 240, 64
+END
+
+/* Proxy Authentication Dialog */
+ID_DLG_PROXY_AUTH DIALOG 29, 23, 154, 65
+STYLE DS_CENTER | WS_POPUP | WS_VISIBLE | WS_CAPTION | DS_CENTER
+CAPTION "OpenVPN - Proxy 인증"
+FONT 8, "Microsoft Sans Serif"
+LANGUAGE LANG_ENGLISH, SUBLANG_DEFAULT
+BEGIN
+    LTEXT "사용자:", 201, 9, 8, 38, 10
+    EDITTEXT ID_EDT_PROXY_USER, 49, 5, 94, 12, ES_AUTOHSCROLL
+    LTEXT "암호:", 202, 9, 26, 38, 10
+    EDITTEXT ID_EDT_PROXY_PASS, 49, 23, 94, 12, ES_PASSWORD | ES_AUTOHSCROLL
+    PUSHBUTTON "확인", IDOK, 58, 43, 40, 14, BS_PUSHBUTTON | WS_TABSTOP | WS_DISABLED
+END
+
+STRINGTABLE
+LANGUAGE LANG_KOREAN, SUBLANG_DEFAULT
+BEGIN
+    IDS_LANGUAGE_NAME "한국어 - Korean"
+
+    /* Tray - Resources */
+    IDS_TIP_DEFAULT "OpenVPN GUI "
+    IDS_TIP_CONNECTED "\n연결됨: "
+    IDS_TIP_CONNECTING "\n연결중: "
+    IDS_TIP_CONNECTED_SINCE "\n연결 시간: "
+    IDS_TIP_ASSIGNED_IP "\n할당된 IP: %s"
+    IDS_MENU_SERVICE "OpenVPN 서비스"
+    IDS_MENU_IMPORT "파일 불러오기…"
+    IDS_MENU_SETTINGS "설정…"
+    IDS_MENU_CLOSE "나가기"
+    IDS_MENU_CONNECT "연결"
+    IDS_MENU_DISCONNECT "연결 해제"
+    IDS_MENU_STATUS "상태 보기"
+    IDS_MENU_VIEWLOG "로그 보기"
+    IDS_MENU_EDITCONFIG "설정 편집"
+    IDS_MENU_PASSPHRASE "암호 변경"
+    IDS_MENU_CLEARPASS  "저장된 암호 삭제"
+    IDS_MENU_SERVICE_START "시작"
+    IDS_MENU_SERVICE_STOP "정지"
+    IDS_MENU_SERVICE_RESTART "재시작"
+    IDS_MENU_SERVICEONLY_START "연결"
+    IDS_MENU_SERVICEONLY_STOP "연결 해제"
+    IDS_MENU_SERVICEONLY_RESTART "재연결"
+    IDS_MENU_ASK_STOP_SERVICE "연결을 해제하겠습니까? (OpenVPN 서비스 중지)"
+
+    /* Logviewer - Resources */
+    IDS_ERR_START_LOG_VIEWER "로그 뷰어 시작 실패: %s"
+    IDS_ERR_START_CONF_EDITOR "설정 편집기 시작 실패: %s"
+
+    /* OpenVPN */
+    IDS_ERR_MANY_CONFIGS "OpenVPN GUI는 %d config 이상의 설정은 지원하지 않습니다. 더이상 설정이 필요한 경우 개발자에게 연락 하십시오."
+    IDS_NFO_NO_CONFIGS "설정 파일이 없습니다.\n"\
+                       """파일 불러오기.."" 메뉴를 이용하거나 설정 파일을 ""%s"" 또는 ""%s""로 복사 하십시오."
+    IDS_ERR_CONFIG_NOT_AUTHORIZED "이 연결(%s)을 시작하기 위하여 ""%s"" 그룹 권한이 필요합니다. \"
+                                  "시스템 관리자에게 문의 하십시오.\n"
+    IDS_ERR_CONFIG_TRY_AUTHORIZE  "이 연결(%s)을 시작하기 위하여 ""%s"" 그룹 권한이 필요합니다.\n\n"\
+                                  "자신을 이 그룹에 추가하겠습니까?\n"\
+                                  "이 작업은 관리자 암호 또는 동의를 요구할 수 있습니다."
+    IDS_NFO_CONFIG_AUTH_PENDING   "이 연결(%s)을 시작하기 위하여 ""%s"" 그룹 권한이 필요합니다.\n\n"\
+                                  "이전의 인증 대화 상자를 완료 하십시오."
+    IDS_ERR_ADD_USER_TO_ADMIN_GROUP """%s"" group에 사용자 추가를 실패했습니다."
+    IDS_ERR_ONE_CONN_OLD_VER "OpenVPN 2.0-beta6 이전 버전에서는 2개 이상의 동시 연결을 지원하지 않습니다."
+    IDS_ERR_STOP_SERV_OLD_VER "You cannot use OpenVPN GUI to start a connection while the OpenVPN Service is running (with OpenVPN 1.5/1.6). Stop OpenVPN Service first if you want to use OpenVPN GUI."
```

Comment:
Missed or deliberately left out? There are a number of such below as well. 
Not everything need to be translated in one attempt, so its perfectly fine if you want to leave these for some other time.